### ISSUE TITLE
Console logs: colorization readability

### ISSUE THREAD

**Issue by tlmii** [dotnet/aspire#1125]
There was some concern brought up during the UX board meeting today about the readability/legibility of some of the text in the console logs when project logs are colorized. Specifically, the default .NET logging "FAIL" display, e.g.:

![image](https://github.com/dotnet/aspire/assets/9613109/f6d449aa-2357-4c44-a99c-94b6d9b5660f)

Right now we are using roughly the same display as windows terminal defaults to (campbell color scheme, 12pt font, cascadia mono font) so we should be pretty close to what they'd see in the actual terminal (rendering differences between the browser and the terminal aside). 

There were some suggestions of trying to use a badge like we do for stderr output since that has better padding and whatnot. 

When thinking about this, we need to remember that it is the customer's code that is deciding what to render here. They could do white-text-on-white-background for instance if they wanted. So anything "custom" we try to do to handle scenarios that don't look great should keep that in mind.

**Comment by adamint** [comment]
I see the concern, however adding a badge is non-trivial. I can tweak the red color for now (the others have sufficient contrast).
